src/services/Fetch/Fetch.ts

```diff
@@ -0,0 +1,55 @@
+import axios from "axios";
+import {
+  LOCAL_FILES_PATHS,
+  REMOTE_DATA_SOURCE_URL,
+} from "../../../utils/constants.js";
+import * as fs from "fs";
+import * as path from "path";
```

Comment:
Si ça te va, on peut se mettre la règle d'importer d'abord les dépendances externes, puis les locales :

```suggestion
import axios from "axios";
import * as fs from "fs";
import * as path from "path";

import {
  LOCAL_FILES_PATHS,
  REMOTE_DATA_SOURCE_URL,
} from "../../../utils/constants.js";
```

Reply:
oui